<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-PKMC2016" ignored="0" name="Login Test" tests="1" failures="1" timestamp="26 Sep 2018 22:02:45 GMT" time="30.754" errors="0">
  <testcase name="loginTest" time="27.767" classname="com.qtpselenium.rediff.hybrid.testcases.portfoliosuite.LoginTest">
    <failure type="java.lang.reflect.InvocationTargetException">
      <![CDATA[java.lang.reflect.InvocationTargetException
	at com.qtpselenium.rediff.hybrid.driver.DriverScript.executeKeywords(DriverScript.java:47)
	at com.qtpselenium.rediff.hybrid.testcases.portfoliosuite.LoginTest.loginTest(LoginTest.java:25)
Caused by: java.lang.AssertionError: The following asserts failed:
	Object Not Found username_id
	at com.qtpselenium.rediff.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:424)
	at com.qtpselenium.rediff.hybrid.keywords.GenericKeywords.getObject(GenericKeywords.java:376)
	at com.qtpselenium.rediff.hybrid.keywords.AppKeywords.login(AppKeywords.java:33)
	... 31 more
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Login Test -->
